packages/broker/src/plugins/operator/VoteOnSuspectNodeService.ts

```diff
@@ -1,15 +1,36 @@
 import { Logger } from '@streamr/utils'
+import StreamrClient from 'streamr-client'
+import { OperatorServiceConfig } from './OperatorPlugin'
+import { VoteOnSuspectNodeHelper } from './VoteOnSuspectNodeHelper'
 
 const logger = new Logger(module)
-
 export class VoteOnSuspectNodeService {
-    // eslint-disable-next-line class-methods-use-this
+    private readonly streamrClient: StreamrClient
+    private readonly voteOnSuspectNodeHelper: VoteOnSuspectNodeHelper
+
+    constructor(streamrClient: StreamrClient, serviceConfig: OperatorServiceConfig) {
+        this.streamrClient = streamrClient
+        this.voteOnSuspectNodeHelper = new VoteOnSuspectNodeHelper(
+            serviceConfig,
+            this.handleNodeInspectionRequest.bind(this)
+        )
+    }
+
     async start(): Promise<void> {
-        logger.info('started')
+        logger.debug('starting')
+        await this.voteOnSuspectNodeHelper.start()
+        logger.debug('started')
     }
 
-    // eslint-disable-next-line class-methods-use-this
     async stop(): Promise<void> {
-        logger.info('stopped')
+        this.voteOnSuspectNodeHelper.stop()
+    }
+
+    async handleNodeInspectionRequest(sponsorship: string, targetOperator: string): Promise<void> {
+        logger.info('Received inspection request', { targetOperator, sponsorship })
+        //const operatorIsMalicious = this.streamrClient.inspectNodes(sponsorship, targetOperato)
+        const operatorIsMalicious = true
+        logger.info(`Vote on inspection request', ${{ sponsorship, targetOperator, kick: operatorIsMalicious }}`)
+        await this.voteOnSuspectNodeHelper.voteOnFlag(sponsorship, targetOperator, operatorIsMalicious)
```

Comment:
Will lead to unhandled promise rejection if `voteOnSuspectNodeHelper.voteOnFlag` rejects because the callback is unhandled synchronously in `VoteOnSuspectNodeHelper`.

I would suggest we turn this into "sync" and catch the error locally as follows:
```suggestion
    handleNodeInspectionRequest(sponsorship: string, targetOperator: string): void {
        logger.info('Received inspection request', { targetOperator, sponsorship })
        //const operatorIsMalicious = this.streamrClient.inspectNodes(sponsorship, targetOperato)
        const operatorIsMalicious = true
        logger.info(`Vote on inspection request', ${{ sponsorship, targetOperator, kick: operatorIsMalicious }}`)
        this.voteOnSuspectNodeHelper.voteOnFlag(sponsorship, targetOperator, operatorIsMalicious).catch((err) => {
            logger.warn('Encountered error while trying to vote on flag', { err })
        })
```